src/schema/sources.ts

```diff
@@ -1252,6 +1258,13 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
             );
           }
 
+          if (args.sortByMembersCount) {
+            builder.queryBuilder.orderBy(
+              `${builder.alias}."membersCount"`,
```

Comment:
this is could be heavy query since we order over `count(*)`, not ideal

Reply:
My thoughts too initially. Maybe we can store it in the flags column then add some triggers. Wdyt?